source/expectations/data_values/ut_compound_data_helper.pkb

```diff
@@ -541,5 +541,169 @@ create or replace package body ut_compound_data_helper is
+                        )           
+                       select distinct :diff_id,tmp.item_hash,tmp.pk_hash,tmp.duplicate_no
+                       from(
+                         (
+                           select t.item_hash,t. duplicate_no,t.pk_hash
+                           from  source_data t
+                           where t.data_id = :self_guid
+                           minus
+                           select t.item_hash,t. duplicate_no,t.pk_hash
+                           from  source_data t
+                           where t.data_id = :other_guid
+                         )
+                           union all
+                         (
+                           select t.item_hash,t. duplicate_no,t.pk_hash
+                           from  source_data t
+                           where t.data_id = :other_guid
+                           minus
+                           select t.item_hash,t. duplicate_no,t.pk_hash
+                           from  source_data t
+                           where t.data_id = :self_guid
+                        ))tmp';
+    return l_sql;
+  end;
+ 
+  function get_inclusion_matcher_sql(a_owner in varchar2) return varchar2 is
+    l_sql varchar2(32767);
+  begin
+    l_sql := 'with source_data as
+                       ( select t.data_id,t.item_hash,t.duplicate_no,
+                           pk_hash
+                           from  ' || a_owner || '.ut_compound_data_tmp t
+                           where data_id = :self_guid or data_id = :other_guid
+                        )           
+                       select distinct :diff_id,tmp.item_hash,tmp.pk_hash,tmp.duplicate_no
+                       from( 
+                         (
+                           select t.item_hash,t. duplicate_no,t.pk_hash
+                           from  source_data t
+                           where t.data_id = :self_guid
+                           minus
+                           select t.item_hash,t. duplicate_no,t.pk_hash
+                           from  source_data t
+                           where t.data_id = :other_guid
+                         )
+                           union all
+                         (
+                           select t.item_hash,t. duplicate_no,t.pk_hash
+                           from  source_data t,
+                           source_data s
+                           where t.data_id = :other_guid 
+                           and s.data_id = :self_guid 
+                           and t.pk_hash = s.pk_hash
+                           and t.item_hash != s.item_hash
+                         )
+                        )
+                        tmp';
+    return l_sql;
+  end;
+   
+   function get_not_inclusion_matcher_sql(a_owner in varchar2) return varchar2 is
+    l_sql varchar2(32767);
+  begin
+    /* Self set does not contain any values from other set */
+    l_sql := 'with source_data as
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: An illegal character with code point 10 was found in this literal. ([plsql:LiteralsNonPrintableCharactersCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3ALiteralsNonPrintableCharactersCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=774&issues=AWbTzUDbO08i52vd_Xcg&open=AWbTzUDbO08i52vd_Xcg)

[ISSUE_KEY]: AWbTzUDbO08i52vd_Xcg